### ISSUE TITLE
 DatePicker styles in list view (list_editable) not working

### ISSUE THREAD

**Issue by hernansaa** [unfoldadmin/django-unfold#686]
**What version of Unfold are you using?**

django-unfold==0.35.0

**What version of Django are you using?**

5.0.6

**What browser are you using?**

Version 127.0.6533.119 (Official Build) (64-bit)

**Did you checked changelog/commit history, if the bug is not already fixed?**

For example: Yes or No

**Did you searched other issues, if the bug is not already fixed?**

Yes

**Describe ISSUE**

When making "list_editable" in the list view, the datepicker style doesnt work properly.  (It does work well in the change_view.)
The problem is with the widget and when you open the calendar modal. The classes of the fields seem correct (see screenshots).
I have tried to use the formfield_overrides (see admin.py code) and without it, but same problem.
(I also tried to user UnfoldAdminDateWidget and AdminDateWidget and the result is incorrect as well but different and not as expected.)

(I am not that experience yet, so I am sorry if there is something I am not doing properly and shouldn't be creating an issue for this.)

Thanks in advance for any help.

LIST VIEW (not working properly)

![list_view_issue](https://github.com/user-attachments/assets/4436e336-f7db-4a06-9868-526487882434)

![Screenshot from 2024-08-18 11-16-39](https://github.com/user-attachments/assets/f0c61a47-38a1-4c8d-a870-580d1c19ead9)

![Screenshot from 2024-08-18 11-23-41](https://github.com/user-attachments/assets/cff21c32-f3d6-4d6c-a2b2-dc1142d1eb46)

CHANGE VIEW (working properly)

![change_view](https://github.com/user-attachments/assets/2d883b2c-2520-48b3-872d-7851a67d3ed0)
![Screenshot from 2024-08-18 11-18-03](https://github.com/user-attachments/assets/ef58d704-2023-4d84-8b8d-da0106c69ab5)
![Screenshot from 2024-08-18 11-22-56](https://github.com/user-attachments/assets/4960aa3c-924a-47f1-8200-68b8207fc206)

# fragment of admin.py

```
@register(Enquiry, site=new_admin_site)
class EnquiryAdmin(ModelAdmin):
    
    compressed_fields = True

    formfield_overrides = {
        models.DateField: {
            "widget": UnfoldAdminSingleDateWidget, 
        },
    }

    list_display = ['student', 'rating','follow_up_date', 'email', 'nationality', 'phone', 'program', 'branch', 'employee', 'created_at']
    list_filter = ['rating', 'program', 'branch', 'employee', 'created_at', 'follow_up_date']
    search_fields = ['name', 'surname', 'email', 'nationality']
    list_per_page = 20
    list_editable = ['branch','employee', 'rating', 'follow_up_date']


    fieldsets = (
        ('Personal Information', {
            'fields': ('name','surname', 'student', 'dob', 'nationality', 'email', 'phone')
        }),
        ('Course Information', {
            'fields': ('program', 'course', 'date_start', 'course_weekly_price', 'course_qty_weeks', 'enrollment_fee',
                       )
        }),
        ('Accommodation', {
            'fields': ('accommodation', 'accommodation_qty_weeks')
        }),
        ('Total', {
            'fields': ('total',)
        }),
        ('Assigment', {
            'fields': ('branch', 'employee')
        }),
        ('Follow Up', {
            'fields': ('rating', 'follow_up_date',)
        }),
        ('Timestamps', {
            'fields': ('created_at',)
        }),
    )

    readonly_fields = (
        'name', 'dob', 'nationality', 'email', 'phone',
        'program', 'course', 'date_start', 'course_weekly_price', 'course_qty_weeks', 'enrollment_fee',
        'accommodation', 'accommodation_qty_weeks', 'total', 'created_at',
    )

```

**Comment by lukasvinclav** [comment]
Thansk for report. Fixed in #715

**Comment by hernansaa** [comment]
> Thansk for report. Fixed in #715

Thanks to you for such a great admin.
